java/spring-boot/src/test/java/benchmark/springboot/BenchmarkApplicationTests.java

```diff
@@ -0,0 +1,43 @@
+package benchmark.springboot;
```

Comment:
Not needed here :stuck_out_tongue:

Reply:
It is a good practice to put the test in the same package as the class under test (but in the test source folder of course).

Reply:
sure, but in this case (a benchmarking one), we do not need them + some functional test are written in `crystal`